rss-client/src/app/Quiz/employee/test-in-progress/test-in-progress.component.html

```diff
@@ -86,7 +99,7 @@ <h4 class="modal-title" id="modal-basic-title">Are you sure?</h4>
       </button>
     </div>
   </ng-template>
-
+<!-- 
   <app-cheater-warning></app-cheater-warning>
-  <app-invalid-quiz></app-invalid-quiz>
+  <app-invalid-quiz></app-invalid-quiz> -->
 </div>
```

Comment:
why is this being commented instead of removed?

Reply:
We did not do this but we can undo it